src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { TodoItem } from '../TodoItem/TodoItem';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filteredTodos: Todo[];
+  onDelete: (id: number) => void;
+  onStatusUpdate: (id: number) => void;
+  onUpdatedTitleSubmit: (
+    arg1: number,
+    arg2: string,
+    arg3: React.Dispatch<React.SetStateAction<boolean>>,
+    arg4: boolean,
+    arg5?: React.FormEvent<HTMLFormElement>,
+  ) => void;
+  onKeyUp: (
+    arg1: React.KeyboardEvent<HTMLInputElement>,
+    arg2: React.Dispatch<React.SetStateAction<boolean>>,
+  ) => void;
+  processedIs: number[];
```

Comment:
```suggestion
  processedIds: number[];
```
typo - fix everywhere